docs/ancestry.md

```diff
@@ -958,17 +958,23 @@ do not result in marginal coalescences. For some purposes, however,
 we want record information on other events of interest, not just the mimimal 
 representation of its outcome.
 
-The `additional_nodes` and `coalescing_segments_only` options serve this exact 
-purpose. These options allow us to record the nodes associated with a custom 
-subset of all events we might observe in the history of a sample. Besides 
-samples and coalescence events, nodes can now also represent common 
-ancestor events, recombination, gene conversion, migration, and pass through 
-events. The example below and the next few paragraphs provide a guide on how 
+The `additional_nodes` and 
+{ref}`coalescing_segments_only <sec_additional_nodes_cso>` options serve 
+this exact purpose. These options allow us to record the nodes associated with 
+a custom subset of all events we might observe in the history of a sample. 
+Besides samples and coalescence events, nodes can now also represent 
+{ref}`common ancestor events <sec_additional_nodes_ca>`, 
+{ref}`recombination <sec_additional_nodes_re>`, 
+{ref}`gene conversion <sec_additional_nodes_re>`, 
+{ref}`migration <sec_ancestry_record_migrations>`, 
+and {ref}`pass through <sec_additional_nodes_re>` events. The example below 
+and the next few paragraphs provide a guide on how 
 to record additional nodes and interpret the resulting node tables.
 
 We first set up a simple pedigree simulation. Pedigrees are an
 interesting starting point as in contrast to the coalescent models a single 
-node might be associated with more than one type of event.
+node might be associated with more than one type of event. Conversely, 
+for coalescent models each event is registered as a new node.
```

Comment:
```suggestion
for the continuous-time coalescent model simulated by default by msprime,
each event is registered as a new node.
```
Couldn't a node in the DTWF be associated with more than one kind of event?

Reply:
yes! good point.